### DISCUSSION TITLE
Change the app accent color

### DISCUSSION THREAD

**Discussion by smellythief** [vincode-io/Zavala#247] [Ideas]
I would like an option to change the app accent color from the current dark green using a color picker.

**Comment by gaelstromm** [comment]
Just want to second & throw my support behind this request. This would be really clutch. 

Even if there were just 4 or 5 preset options or an option to make it neutral… 

The green is not *horrible*—it does have a very strong/characteristic hue however. Some people have acute aversion to specific color frequencies— beyond aesthetics / color schemes, it can be a both a cognitive & visual accessibility issue for some people. So having the option to at least swap between high contrast / neutral accent for example can go a long way. 

(For example.. Native iOS apple apps have much farther to go on their accessibility UIs to the point it can force people off of apple devices all together. The green is not anywhere near as atrocious or maddening as iOS Note’s forced yellow & then the Calendar’s red that are somehow as immutable as not being able to place homescreen apps in hovered spaces, but just as a point of reference  it’s a big enough issue for me personally with those apps that I cannot use them at all.. don’t get me started on the clock UI 🤦🏻. Those are way worse obviously bc they are integrated into the iOS system itself & can’t be replaced the same way by 3rd party apps. I’m sure for some ppl it’s minor, for others it can be essential.)

**Comment by vincode-io** [comment]
On macOS there is an Accent Color option in Appearance in the System Settings. It allows you to select an Accent Color that works system wide for all apps that support the setting. (Zavala has support for this.) I have no idea why this doesn't also exist on iOS yet.

Given that Apple gave us a way to color icons in the last iOS release, I'm going to hold off implementing anything with the hopes that they support a system wide accent color selection in the next release. I'd hate to implement something custom and then have Apple turn around and give us a sanctioned solution to this problem.

**Reply by mjhaxby** [reply]
Hello. For me, The macOS accent colour setting is overridden on my Intel 2020 MacBook Air (where the system-wide accent colour is pink, but this gets mixed with Zavala's green), but not on my Mac Studio (which correctly respects the accent colour as yellow). I can open a new issue if you think it's worth it.
![Image 20-02-2025 at 11 43](https://github.com/user-attachments/assets/32c50bfe-e646-41ec-8559-f2e69e36f4f7)

**Reply by smellythief** [reply]
> Given that Apple gave us a way to color icons in the last iOS release, I'm going to hold off implementing anything with the hopes that they support a system wide accent color selection in the next release.

Apple is so much more controlling of iOS than macOS that I seriously doubt that they will do this anytime soon. Although they are moving in the more lenient direction, they move glacially slow still. It will be 5+yrs for sure before they give you this.

**Reply by vincode-io** [reply]
@mjhaxby That is weird. Could you please file a bug report for that and include a screenshot of your Appearance settings in System Settings?

**Comment by vincode-io** [comment]
@smellythief One thing I am working on right now is keyboard navigation for the main window. When I'm done the only time the accent color will be used will be when you are actively selecting in a Tag or an Outline. Otherwise a neutral color will show which Tag or Outline is selected, so in the future the accent color will be much less prominent while editing an Outline. This should help your situation a bit.
